docker tests are failing

`test_log_system_probe_unconfigured` seems to be failing.

pinging @mikemhenry

I will have to grok this some more, it started awhile ago 
![image](https://github.com/OpenFreeEnergy/openfe/assets/11765982/93bfd31e-f842-48a0-b69c-d340de9e6cbb)

```
     def test_log_system_probe_unconfigured():
        # if probe loggers aren't configured to run, then we shouldn't even call
        # _probe_system()
        logger_names = [
            'openfe.utils.system_probe.log',
            'openfe.utils.system_probe.log.gpu',
            'openfe.utils.system_probe.log.hostname',
        ]
        # check that initial conditions are as expected
        for logger_name in logger_names:
            logger = logging.getLogger(logger_name)
>           assert not logger.isEnabledFor(logging.DEBUG)
E           assert not True
E            +  where True = <bound method Logger.isEnabledFor of <Logger openfe.utils.system_probe.log (DEBUG)>>(10)
E            +    where <bound method Logger.isEnabledFor of <Logger openfe.utils.system_probe.log (DEBUG)>> = <Logger openfe.utils.system_probe.log (DEBUG)>.isEnabledFor
E            +    and   10 = logging.DEBUG

/opt/conda/lib/python3.10/site-packages/openfe/tests/utils/test_system_probe.py:455: AssertionError
```

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -83,7 +83,7 @@ jobs:
         run: |
           pytest -n auto -v --cov=openfe --cov=openfecli --cov-report=xml --durations=10
 
-      - name: codecov
+      - name: codecov-pr
         if: ${{ github.repository == 'OpenFreeEnergy/openfe'
                 && github.event_name == 'pull_request' }}
         uses: codecov/codecov-action@v3
@@ -92,3 +92,15 @@ jobs:
           file: coverage.xml
           fail_ci_if_error: False
           verbose: True
+          flags: fast-tests
+
+      - name: codecov-schedule
+        if: ${{ github.repository == 'OpenFreeEnergy/openfe'
+                && github.event_name == 'schedule' }}
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          file: coverage.xml
+          fail_ci_if_error: False
+          verbose: True
+          flags: slow-tests

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -95,9 +95,12 @@ jobs:
           flags: fast-tests
 
       - name: codecov-merge
-        if: ${{ github.repository == 'OpenFreeEnergy/openfe'  # we only want to upload a slow report if
-                && github.event_name != 'schedule'            # 1) it isn't a schedule run
-                && github.event_name != 'pull_request' }}     # 2) it wasn't from a PR (we don't run slow tests on PRs)
+        # we only want to upload a slow report if
+        # 1) it isn't a schedule run
+        # 2) it wasn't from a PR (we don't run slow tests on PRs)
+        if: ${{ github.repository == 'OpenFreeEnergy/openfe'  
+                && github.event_name != 'schedule'            
+                && github.event_name != 'pull_request' }}     
         uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -95,8 +95,9 @@ jobs:
           flags: fast-tests
 
       - name: codecov-schedule
-        if: ${{ github.repository == 'OpenFreeEnergy/openfe'
-                && github.event_name == 'schedule' }}
+        if: ${{ github.repository == 'OpenFreeEnergy/openfe'  # we only want to upload a slow report if
+                && github.event_name != 'schedule'            # 1) it isn't a schedule run
+                && github.event_name != 'pull_request' }}     # 2) it wasn't from a PR (we don't run slow tests on PRs)
         uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -94,7 +94,7 @@ jobs:
           verbose: True
           flags: fast-tests
 
-      - name: codecov-schedule
+      - name: codecov-merge
         if: ${{ github.repository == 'OpenFreeEnergy/openfe'  # we only want to upload a slow report if
                 && github.event_name != 'schedule'            # 1) it isn't a schedule run
                 && github.event_name != 'pull_request' }}     # 2) it wasn't from a PR (we don't run slow tests on PRs)

--- a/.github/workflows/docker-cron.yaml
+++ b/.github/workflows/docker-cron.yaml
@@ -90,4 +90,4 @@ jobs:
           docker run --rm ${{ steps.fqirp.outputs.FQIRP }} openfe --version
           docker run --rm ${{ steps.fqirp.outputs.FQIRP }} python -c "import gufe; print(gufe.__version__)"
           docker run --rm ${{ steps.fqirp.outputs.FQIRP }} pytest --pyargs gufe -v
-          docker run --rm ${{ steps.fqirp.outputs.FQIRP }} pytest --pyargs openfecli --pyargs openfe -v
+          docker run --rm ${{ steps.fqirp.outputs.FQIRP }} pytest --pyargs openfe --pyargs openfecli -v

